src/pages/GetStarted.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ProfileStepper from '../components/ProfileStepper';
```

Comment:
These changes shouldn't be a part of this PR

Reply:
Ok got it this was removed from this PR